client/components/details-link/test/index.test.tsx

```diff
@@ -19,16 +19,9 @@ describe( 'Details link', () => {
 		expect( link ).toMatchSnapshot();
 	} );
 
-	test( 'renders dispute details with ID', () => {
-		const { container: link } = render(
-			<DetailsLink id="dp_mock" parentSegment="disputes" />
-		);
-		expect( link ).toMatchSnapshot();
-	} );
-
 	test( 'empty render with no ID', () => {
 		const { container: link } = render(
-			<DetailsLink parentSegment="disputes" />
+			<DetailsLink parentSegment="transactions" />
```

Comment:
I wonder if we could implement this change without needing to tweak the tests for `DetailsLink`. What would that look like? Is it possible?

Reply:
(again, not on the critical path for this PR)

Reply:
Removed changes to `DetailsLink` in 2e47bb83b8d0a85deecfb658622d29be1b8b44eb.